app/services/cocina_generator/structural/generator.rb

```diff
@@ -54,7 +54,8 @@ def get_preserved_file(filename)
       # rubocop:disable Metrics/AbcSize
       def build_fileset(attached_file:, fileset: nil)
         # h2 only has one file per fileset
-        cocina_file = fileset&.structural&.contains&.first
+        structural = fileset&.structural
+        cocina_file = structural&.contains&.first
```

Comment:
This is to address the new `Style/SafeNavigationChainLength` cop without just doing a disable on the line. If we'd prefer the disable, I'm happy to take that approach as well.